node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -77,6 +78,10 @@ impl<I: NodeIdT, C: Context> PendingVertices<I, C> {
         self.0.retain(|pvv, _| pvv.inner().is_evidence());
     }
 
+    pub(crate) fn len(&self) -> u64 {
+        self.0.len() as u64
+    }
```

Comment:
This is the number of pending vertices, counting each only once, even if there are multiple peers.
Would it make sense to get the number of peer-vertex pairs as well, to see if we have unreasonably many sender per vertex on average?

Reply:
I did that at the beginning but I think it's misleading b/c each vertex should be downloaded once.